squidDrone/tests/imu/imu_interface_tests.cpp

```diff
@@ -1,17 +1,17 @@
 #include "gtest/gtest.h"
-#include "i2c_handler.hpp"
+#include "i2c.hpp"
```

Comment:
Is this dependency a mock? If not make it a mock. In unittesting only the unit under test may be the original code (imu in this case) and everything else has to be mocked (except language features, standard libraries and types that are never amiguous (structs without functionality, enum, etc))

Reply:
The Headerfile is the original Headerfile from this component. But the implementation is mocked. Atm. only the Interface is used here, so a unique_ptr on I2C is created, but no implementation used so far. With next development step in IMU, the implementation of I2C will be mocked for further IMU testing.

Reply:
Ok. In this case this is adequate.